### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Common questions_ColorContrast: Color contrast ratio below 3:1 for the edit field present at the bottom of the page.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3597]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.4.11;#Win10;#Visual Studio Code;#CELAP1_code;#ColorContrast;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/versioncontrol
2)Using Version Control in VS Code Page will get open.
3)Go through "Common questions" heading till the feedback section.
4)Select yes/no button and then check for the color contrast using Accessibility Insights for Desktop.

**Actual:**
The color contrast of the edit field present in the feedback section is below 3:1 ratio.

Similar issue has been observed in following pages:
1.https://code.visualstudio.com/docs/editor/whyvscode
2.https://code.visualstudio.com/docs/getstarted/settings 
3.https://code.visualstudio.com/docs/python/python-tutorial
4.https://code.visualstudio.com/docs/getstarted/tips-and-tricks
5.https://code.visualstudio.com/docs/setup/linux

**Expected:**
The expected color contrast ratio of the edit field should be equal to or greater than 3:1 ratio to be clearly visible for low vision users.

**User Impact:**
Low vision users might not be able to clearly see the edit field as the color contrast ratio for the borders of the edit field is below 3:1 ratio.

**Attachment link for Reference:**
[3597_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_Common.questions_ColorContrast.zip](https://github.com/microsoft/vscode-docs/files/4537633/3597_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_Common.questions_ColorContrast.zip)

**Comment by gregvanl** [comment]
Won't fix as the detecting the border is not required to enter text in the feedback control.

**Comment by v-atkulk** [comment]
@gregvanl as per the discussion with SME, the border for the edit field for entering text of feedback is required. Hence request you to please reopen the bug.

**Comment by gregvanl** [comment]
The contrast between the background and text box is high 21:1 which should be sufficient. The halo and unfocused gray outline aren't required to use the text box.

![image](https://user-images.githubusercontent.com/12818376/82615208-b7cbe900-9b7e-11ea-8bde-ba2b6e136438.png)

**Comment by ManishaC1** [comment]
Hello @gregvanl 
We need to manually sample the color in order to get a good result as Auto-detect contrast ratio will not yield a good result. The issue is still reproducible on all the URLs mentioned above. Attached is the repro attachment.
![Screenshot (435)](https://user-images.githubusercontent.com/105198219/217169642-5166ce27-c9b9-44ca-bc3b-c4d94126fcf1.png)
